ArcGISRuntimeSDKQt_CppSamples/DisplayInformation/ReadSymbolsFromMobileStyle/ReadSymbolsFromMobileStyle.h

```diff
@@ -79,8 +79,8 @@ class ReadSymbolsFromMobileStyle : public QObject
   SymbolImageProvider* m_symbolImageProvider = nullptr;
   QList<Esri::ArcGISRuntime::SymbolStyleSearchResultListModel*> m_models = { nullptr, nullptr, nullptr, nullptr };
   QList<QUuid> m_taskIds;
-  QColor m_currentColor;
-  int m_symbolSize;
+  QColor m_currentColor = QColor("yellow");
```

Comment:
Use the enum color constructor. This is one of the clazy warnings that will come up. 

Be careful, however. Make sure that the "yellow" string corresponds with `Qt::yellow` enum. That is not is obvious is it should be.
`QColor m_currentColor = QColor(Qt::yellow)`

Reply:
seems to work well. thanks